deploy/roles/zeppelin/tasks/main.yml

```diff
@@ -0,0 +1,40 @@
+# main.yml - Zeppelin installation
+---
+- name: Ensure Zeppelin source directory exists
+  file:
+    path: "{{ zeppelin_src_dir }}"
+    state: directory
+    mode: 0755
+  tags:
+    - zepplin 
+
+- name: Download zeppelin binary
+  get_url:
+    url: "{{ zeppelin_url }}"
+    dest: "{{ zeppelin_src_dir }}"
+  tags:
+    - zeppelin 
+
+- name: Extract zepplin 
+  unarchive:
+    src: "{{ zeppelin_gz_file }}"
+    dest: "{{ zeppelin_parent_dir }}"
+    copy: no 
+  tags:
+    - zeppelin 
```

Comment:
@olivermel See my comments on the Beaker PR, same thing applies here in terms of using just the "unarchive" command to both download and extract the compressed archive.